assets/src/edit-story/app/snackbar/snackbarProvider.js

```diff
@@ -18,56 +18,38 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
-import { useCallback, useMemo, useRef, useState } from 'react';
+import { useCallback, useMemo, useState } from 'react';
 import { v4 as uuidv4 } from 'uuid';
 /**
  * Internal dependencies
  */
 import Context from './context';
 import SnackbarContainer from './snackbarContainer';
 
-function SnackbarProvider({ children, place }) {
+function SnackbarProvider({ children, place = 'bottom-left' }) {
```

Comment:
Would it be possible to just have 1 SnackbarProvider in design system that's used by both dashboard and editor? Or are they so drastically different?

It just seems like a good opportunity to remove a ton of duplicated code.

Reply:
This change is a little more extensive but I think it would be good to address right now. The snackbar contexts use different data structures (array in Editor and object in Dashboard) and some other slight differences. The differences are enough that it will probably take me longer to test and assure I'm not breaking anything if I include the changes.

@swissspidy What I'm gonna do for expediency is make a follow-up PR to this one to pay this debt while this one moves ahead to be ready for bug bash.